bibtex-actions.el

```diff
@@ -169,6 +169,32 @@ and nil means no action."
   :group 'bibtex-actions
   :type 'function)
 
+(defcustom bibtex-actions-major-mode-functions
+  '(((latex-mode) .
```

Comment:
Should this be a list of major-modes as here or a function so that something like `derived-mode-p` can be used? The later choice seems more flexible. So we can just say `(derived-mode-p 'latex-mode)` instead of having to enumerate the derived ones.